docs/python/examples/README.txt

```diff
@@ -0,0 +1,5 @@
+Gallery of examples
```

Comment:
is this file needed?

Reply:
yes, Sphinx requires a README in `examples/` as well to build successfully.